gap/basic/pcpgrps.gi

```diff
@@ -163,7 +163,7 @@ InstallMethod( IsNormalOp, "for pcp groups",
                IsIdenticalObj, [ IsPcpGroup, IsPcpGroup ],
 function( H, U )
     local u, h;
-    for h in GeneratorsOfPcp( Pcp(H, U)) do
+    for h in GeneratorsOfPcp( Pcp(H)) do
```

Comment:
That makes sense.

But taking a step back, I wonder whether why we bother installing this method *at all*; the generic method should be just fine, no?
```
InstallMethod( IsNormalOp,
    "generic method for two groups",
    IsIdenticalObj, [ IsGroup, IsGroup ],
    function( G, H )
    return ForAll(GeneratorsOfGroup(G),
             i->ForAll(GeneratorsOfGroup(H),j->j^i in H));
    end );
```
I mean, using `Pcp` and `Igs` could in theory be "more efficient" if a group happens to have a ton of redundant generators set; but likewise it could be "less efficient" if the group has fewer generators set.

Reply:
I don't see a (good) reason not to use the generic method either. I suppose removing (or commenting out) the method is the best course of action?

Reply:
I would just remove it. If we later find that it caused a regression somewhere, we can still revert this (everything is in the repo history after all)

Reply:
Done!